common-tasks/backup-restore.md

```diff
@@ -11,15 +11,45 @@ redirect_from:
 Qubes Backup, Restoration, and Migration
 ========================================
 
-**Caution:** The Qubes backup system currently relies on a [weak key derivation scheme](https://github.com/QubesOS/qubes-issues/issues/971). It is *strongly recommended* that users select a *high-entropy* passphrase for use with Qubes backups.
+**Caution:** The Qubes R3.2 backup system currently relies on a [weak key derivation scheme](https://github.com/QubesOS/qubes-issues/issues/971). Although resolved in R4.0 and higher with the switch to scrypt, it is *strongly recommended* that users select a *high-entropy* passphrase for use with Qubes backups.
 
 
 With Qubes, it's easy to back up and restore your whole system, as well as to migrate between two physical machines.
 
 As of Qubes R2B3, these functions are integrated into the Qubes VM Manager GUI. There are also two command-line tools available which perform the same functions: [qvm-backup](/doc/dom0-tools/qvm-backup/) and [qvm-backup-restore](/doc/dom0-tools/qvm-backup-restore/).
 
 
-Creating a Backup
+Creating a Backup (R4.0 and later)
+-----------------
+
+1. In **Qubes VM Manager**, click **System** on the menu bar, then click **Backup Qubes** in the drop-down list. This brings up the **Qubes Backup VMs** window.
+
+2. Move the VMs that you want to back up to the right-hand **Selected** column. VMs in the left-hand **Available** column will not be backed up.
+
+   **Note:** A VM must be shut down in order to be backed up. Currently running VMs appear in red.
+
+   You may choose whether to compress backups by checking or unchecking the **Compress the backup** box. Normally this should be left on unless you have a specific reason otherwise.
+   
+   Once you have selected all desired VMs, click **Next**.
+
+3. Select the destination for the backup:
+
+   If you wish to send your backup to a (currently running) VM, select the VM in the drop-down box next to **Target AppVM**.
+   If you wish to send your backup to a [USB mass storage device](/doc/stick-mounting/), first mount the device in a VM, then select the mount point inside that VM as the backup destination.
```

Comment:
Not sure if worth mentioning, but directory selection widget allow you to mount connected device, under "Other locations" item on the left.